frontend/src/graphql/mutations.js

```diff
@@ -73,7 +73,13 @@ export const createTransactionLink = gql`
 `
 
 export const deleteTransactionLink = gql`
-  mutation($id: Float!) {
+  mutation($id: Int!) {
```

Comment:
Same question as for the redeem mutation, I think that it would be safer if we use the code and not the id of the transactionLink